test/runtests.jl

```diff
@@ -147,13 +147,17 @@ end
 @testset "Divisors" begin
     # Compute properties of toric divisors on Hirzebruch surface
     D=ToricDivisor( [ 0,0,0,0 ], H5 )
+    JToric.show( D )
     @test JToric.iscartier( D ) == true
     @test JToric.isprincipal( D ) == true
     @test JToric.is_primedivisor( D ) == false
     @test JToric.isbasepoint_free( D ) == true
     @test JToric.isample( D ) == false
     @test JToric.isvery_ample( D ) == false
     @test JToric.isnef( D ) == true
+    @test JToric.isintegral( D ) == true
+    @test JToric.isq_cartier( D ) == true
+    @test JToric.polyhedron_of_divisor( D ) isa FOOBAR
```

Comment:
That won't work, you need to replace FOOBAR with something sensible -- I did not know what is returned, so I inserted a placeholder ;-).